requirements.txt

```diff
@@ -1,5 +1,16 @@
-requests==2.20.0
-Flask==1.0.2
 beautifulsoup4==4.6.3
```

Comment:
I've a feeling that the requirements.txt includes dependencies installed in your environment and not just those that are required for `wimp`. Try with virtual environment once.

Reply:
Sure would take a look at this.

Reply:
Doesnt look like any of my system dependencies are there, Used  pipenv this time for a virtual env.